performance-comparisons/v11.8/results/report_0_postgresql_bounded_6500.md

```
@@ -0,0 +1,785 @@
+| ScheduledPostStore.GetPendingScheduledPosts | p99 | 5ms | 9ms | 4ms | 80.81
+| UserStore.GetMany | p99 | 5ms | 9ms | 4ms | 80.81
+| StatusStore.SaveOrUpdateMany | p99 | 25ms | 43ms | 18ms | 72.56
+| RoleStore.ChannelHigherScopedPermissions | p99 | 5ms | 8ms | 3ms | 60.61
+| EmojiStore.GetByName | p99 | 5ms | 8ms | 3ms | 57.16
+| PostStore.SearchPostsForUser | p99 | 670ms | 964ms | 294ms | 43.88
+| ChannelStore.CreateDirectChannel | p99 | 50ms | 64ms | 14ms | 28.17
+| ChannelStore.GetPublicChannelsForTeam | p99 | 25ms | 28ms | 3ms | 12.00
+### Store times avg (improved):
+| | | Base | Actual | Delta | Delta % |
+| --- | --- | --- | --- | --- | --- |
+| ChannelBookmarkStore.Get | avg | 2ms | 0s | -2ms | -117.64
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)